CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.2)
+
+project(omim C CXX)
+
+add_compile_options(
+  "-Wall"
+  "-pedantic"
+  "-std=c++11"
+)
+
+
+find_package (Threads)
+
+#add_definitions(-DDEBUG)
+INCLUDE_DIRECTORIES(${CMAKE_HOME_DIRECTORY})
+
+subdirs(base)
+
+message("BUILD TYPE : >>${CMAKE_BUILD_TYPE}<<")
```

Comment:
I guess you can remove this line now. Or move it to the end of the file, because CMAKE_BUILD_TYPE is modified later.